.github/workflows/build-container-image.yml

```diff
@@ -76,28 +65,108 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: docker/metadata-action@v5
+      - name: Docker meta
         id: meta
+        uses: docker/metadata-action@v5
         if: ${{ inputs.push_to_images != '' }}
         with:
           images: ${{ inputs.push_to_images }}
           flavor: ${{ inputs.flavor }}
-          tags: ${{ inputs.tags }}
           labels: ${{ inputs.labels }}
 
-      - uses: docker/build-push-action@v6
+      - name: Build and push by digest
+        id: build
+        uses: docker/build-push-action@v6
+        if: ${{ inputs.push_to_images != '' }}
```

Comment:
If this is disabled when `push_to_images` is empty, it means the container is never built when `test-image-build.yml` runs? Only the push should be disabled in this case?